lib/howl/commands/file_commands.moon

```diff
@@ -50,14 +53,19 @@ command.register
 command.register
   name: 'save',
   description: 'Saves the current buffer to file'
+  evade_history: true
   handler: ->
     buffer = app.editor.buffer
-    return command.run 'save-as' unless buffer.file
+    if not buffer.file
+      app.window.command_line\run_after_finish ->
+        command.run 'save-as' unless buffer.file
```

Comment:
The `unless buffer.file` seems redundant given that it's already being checked above, or is it because that could change before the invocation?